shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/viewmodel/FeedingsState.kt

```diff
@@ -6,5 +6,7 @@ import kotlinx.collections.immutable.persistentListOf
 
 data class FeedingsState(
     val feedings: ImmutableList<FeedingModel> = persistentListOf(),
-    val isLoading: Boolean = false
+    val feedingsFiltered: ImmutableList<FeedingModel> = persistentListOf(),
```

Comment:
Let's just use `feedings` field

Reply:
Forgot to add that you can store all feeding in the viewmodel, but not expose it to UI

Reply:
I was thinking in maybe keeping a cache in some Repository, similar to `FeedingPointRepositoryImpl`, but that would require some refactor. I'll keep it at the view model then.